runware/types.py

```diff
@@ -144,140 +144,98 @@ class IRefiner:
     guide_image: Optional[Union[str, File]] = None
     guide_image_unprocessed: Optional[Union[str, File]] = None
 
-
-@dataclass
-class IControlNetA(IControlNetGeneral):
     def __post_init__(self):
-        # This method will run after the default __init__ generated by @dataclass
+        self.validate_guide_images()
+
+    def validate_guide_images(self):
         guide_images = [self.guide_image, self.guide_image_unprocessed]
-        # Check that exactly one of the fields is provided
         if sum(x is not None for x in guide_images) != 1:
             raise ValueError(
                 "Exactly one of 'guide_image' or 'guide_image_unprocessed' must be provided, not both or none."
             )
 
 
 @dataclass
-class IControlNetCanny:
-    weight: Optional[float]
-    start_step: Optional[int]
-    end_step: Optional[int]
-    control_mode: Optional[EControlMode]
-    model: Union[int, str]
-    low_threshold_canny: Optional[int]
-    high_threshold_canny: Optional[int]
-    preprocessor: EPreProcessor = EPreProcessor.canny
-    guide_image: Optional[Union[str, File]] = None
-    guide_image_unprocessed: Optional[Union[str, File]] = None
-    include_hands_and_face_open_pose: bool = field(default=True)
-
+class IControlNetA(IControlNetGeneral):
     def __post_init__(self):
-        # This method will run after the default __init__ generated by @dataclass
-        guide_images = [self.guide_image, self.guide_image_unprocessed]
-        # Check that exactly one of the fields is provided
-        if sum(x is not None for x in guide_images) != 1:
-            raise ValueError(
-                "Exactly one of 'guide_image' or 'guide_image_unprocessed' must be provided, not both or none."
-            )
+        super().__post_init__()
 
 
 @dataclass
-class IControlNetHandsAndFace:
-    preprocessor: EOpenPosePreProcessor
-    include_hands_and_face_open_pose: bool
-    weight: Optional[float]
-    start_step: Optional[int]
-    end_step: Optional[int]
-    control_mode: Optional[EControlMode]
-    guide_image: Optional[Union[str, File]] = None
-    guide_image_unprocessed: Optional[Union[str, File]] = None
+class IControlNetCanny(IControlNetGeneral):
+    low_threshold_canny: Optional[int] = None
+    high_threshold_canny: Optional[int] = None
+    preprocessor: EPreProcessor
 
     def __post_init__(self):
-        # This method will run after the default __init__ generated by @dataclass
-        guide_images = [self.guide_image, self.guide_image_unprocessed]
-        # Check that exactly one of the fields is provided
-        if sum(x is not None for x in guide_images) != 1:
-            raise ValueError(
-                "Exactly one of 'guide_image' or 'guide_image_unprocessed' must be provided, not both or none."
-            )
+        super().__post_init__()
 
 
-class IControlNetBaseWithUUID(ABC):
-    pass
+@dataclass
+class IControlNetHandsAndFace(IControlNetGeneral):
+    include_hands_and_face_open_pose: Optional[bool] = None
```

Comment:
camel case - but also please be sure to entirely match remote parameter names, this one seems absurd, is it really what's on the remote side? not our fault, just wondering..